### ISSUE TITLE
Dynamic import plugin

### ISSUE THREAD

**Issue by wilcoxmd** [i18next/i18next#2072]
## Documentation issue

👋 Hi! 

Is it recommended to use the [`i18next-resources-to-backend`](https://github.com/i18next/i18next-resources-to-backend) package as a standalone backend plugin that can use dynamic import? If so, it would be nice to clarify some of the docs. 

I've been a bit confused about whether I should use `i18next-resources-to-backend` for the following reasons: 

* It's not listed under the i18next [plugin list](https://www.i18next.com/overview/plugins-and-utils). 
* Instead, there's just a link to a [webpack import plugin](https://gist.github.com/SimeonC/6a738467c691eef7f21ebf96918cd95f) gist. 
* The `i18next-resources-to-backend` package's readme only shows more complex usage with chained backends, and the comments are a bit hard to follow. Some of the questions I had: 
    * Do I need to consider using the `HttpBackend` as well?
    * Do I need to do something to enable the `inMemoryLocalBackend`?
    * Is a `loadPath` required if just relying on dynamic imports? 

This all makes me unsure when I should use i18next-resources-to-backend, and what the official support level of the package is today by the i18next team.

**Comment by adrai** [comment]
- just added to the plugin list and extended the readme a bit (feel free to suggest more docs improvements with a PR if you like to)
- the gist you shared is basically a very light user-land solution to what i18next-resources-to-backend is
- you don't need to use the i18next-http-backend as well
- the inMemoryLocalBackend are the imported/provided resources via i18next-resources-to-backend
- no a loadPath is not required if just relying on dynamic imports a simple example is also shown here: https://www.i18next.com/how-to/add-or-load-translations#lazy-load-in-memory-translations
- and yes, i18next-resources-to-backend is officially supported by us

I hope this info helps.

**Comment by wilcoxmd** [comment]
Thanks! That is all very helpful. 🙏 

One more clarification: is there any reason to use the example set by the gist vs what is provided by `i18next-http-backend`?

**Comment by adrai** [comment]
Like said before the gist is not official... I would use the i18next-resources-to-backend only.
But choosing to use i18next-http-backed or i18next-resources-to-backend (or both via i18next-chained-backend) depends on your needs.
If you're hosting the translation resources json on a web server or plan to do so, then choose the i18next-http-backend.
If you will always only use in-memory translations you can use i18next-resources-to-backend or also directly use the [resources option](https://www.i18next.com/how-to/add-or-load-translations#add-on-init). If you like to dynamically import the translations use i18next-resources-to-backend.
If you need some sort of fallback handling use both via i18next-chained-backend, like [here](https://www.i18next.com/how-to/backend-fallback#browser-fallback-with-local-bundled-translations).

**Comment by wilcoxmd** [comment]
Thanks, that clears things up. I appreciate the help!
